### DISCUSSION TITLE
Glitchy banner drop shadow in iOS Safari

### DISCUSSION THREAD

**Discussion by asmeurer** [pradyunsg/furo#415] [Potential Issue]
In iOS Safari, when I go to, e.g., https://pradyunsg.me/furo/kitchen-sink/blocks/, the drop shadow of the top banner glitches in and out as I scroll. Sometimes when I stop scrolling it is there, and sometimes it is not. I can record a video showing what happens if you'd like. 

This might be a bug in mobile Safari as I cannot reproduce it in desktop Chrome, Safari, or Firefox.  However, even on desktop, I noticed that the drop shadow is not there when I initially load the page, but when I scroll down it appears, and doesn't disappear again when I scroll back to the top of the page. It's not clear to me what the banner drop shadow behavior is supposed to be in any case. 

I am on the latest iOS and macOS.

**Comment by pradyunsg** [comment]
Bundling that in with https://github.com/pradyunsg/furo/issues/681
